### PR TITLE
Show .exists status of pointer in its __repr__

### PR DESCRIPTION
**Pull request by shubham3121**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

- create pointer status enums
- refactor __repr__ for pointer, GammaTensorWrappedPointer, PhiTensorWrappedPointer
- include pointer exist status in __repr__ of the respective Pointer classes

Fixes https://github.com/OpenMined/PySyft/issues/6935

Screenshot from 2022-11-23 19-42-30

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: packages/syft/src/syft/core/node/enums.py [hunk 1]
```diff
@@ -37,3 +40,8 @@ class ResponseObjectEnum(str, Enum):
     USER = "user"
     WORKER = "worker"
     DATA = "data"
+
+
+class TensorPointerStatus(str, Enum):
```
Comment (rasswanth-s) [inline packages/syft/src/syft/core/node/enums.py:45]: MAYBE: we could keep as PointerStatus, as we use them with several pointers , like phi,gamma also?

File: packages/syft/src/syft/core/node/enums.py [hunk 2]
```diff
@@ -37,3 +40,8 @@ class ResponseObjectEnum(str, Enum):
...
+    READY = bcolors.green("Ready")
+    PROCESSING = bcolors.yellow("Processing")
```
Comment (rasswanth-s) [inline packages/syft/src/syft/core/node/enums.py:47]: Could also add them to packages/syft/core/tensor.py --> `TensorPointer`

**Comment by rasswanth-s** [comment]
Yayy! The output is so nice!!!!

**Review by rasswanth-s — APPROVED** [review]
Well Done @shubham3121 left some minor comments

File: packages/syft/src/syft/core/node/enums.py (hunk 2 above)
Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/enums.py:47]: Solved by: [8164bae](https://github.com/OpenMined/PySyft/pull/7105/commits/8164bae204debcfdeaf15d377c61bf68f7f5ef2b)

File: packages/syft/src/syft/core/node/enums.py (hunk 1 above)
Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/enums.py]: Solved by: [8164bae](https://github.com/OpenMined/PySyft/pull/7105/commits/8164bae204debcfdeaf15d377c61bf68f7f5ef2b)

File: packages/syft/src/syft/core/node/enums.py (hunk 2 above)
Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/enums.py:47]: TensorPointer doesn't implement `__repr__` making him use his parent (_Pointer_) __repr__ method. 
tensor_pointer_ready

**Pull request merged**